contents/blog/startup-ops-toolkit.md

```diff
@@ -1,102 +1,133 @@
 ---
-date: 2021-03-23
-title: The startup ops toolkit
+date: 2022-05-03
+title: The PostHog ops toolkit for early-stage startups
 rootPage: /blog
 sidebar: Blog
 showTitle: true
 hideAnchor: true
 author: ["charles-cook"]
-categories: ["Startups", "Inside PostHog"]
+featuredImage: ../images/blog/posthog-release-notes-blog.png
+featuredImageType: full
+categories: ["Startups", "Inside PostHog", "Guides"]
 ---
 
 Ok, so you’ve read *Zero to One* and *The Hard Thing About Hard Things*, but what are all the boring admin things that are actually going to drain 40% of your time which no one has told you about?
 
-<iframe src="https://giphy.com/embed/j6wvxJLueHV10lATCF" width="480" height="360" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/usnationalarchives-archivesgif-youinpublicservice-publicservice-j6wvxJLueHV10lATCF">via GIPHY</a></p>
+More importantly, what is the ops stack you can use to streamline and automate these?
 
-More importantly, what are the tools you can use to streamline and automate these?
+**TL;DR:** It’s [Deel](https://www.letsdeel.com/) + [Gusto](https://gusto.com/) + [Fondo](https://www.tryfondo.com/) + [Pry](https://pry.co/) + [Brex](https://www.brex.com/) + [some free contract templates](https://www.orrick.com/en/Total-Access/Tool-Kit/Start-Up-Forms/Employment-and-Consultant + [Google Sheets/Docs](https://docs.google.com/) + [Workable](https://www.workable.com/) + [Hubspot](https://www.hubspot.com/) + [CharlieHR](https://www.charliehr.com/). 
```

Comment:
Maybe add Anrok to this for managing tax liability under the managing money section?